app/main.py

```diff
@@ -275,8 +307,8 @@ def manage_login_modal(n_clicks, username, password):
         if not check:
             # This if statement is verified if credentials are invalid
 
-            # We add the appropriate feedback
-            form_feedback.append(html.P("Nom d'utilisateur ou mot de passe erroné."))
+            # We make the HTTP request to the login route of the API
+            response = requests.post(login_route_url, data=data).json()
```

Comment:
I think this was done by mistake, right? The POST request is done before the `if not check:` line.

Don't worry with this anyway, I will include this in my PR! 

(And do let me know if I am missing something of course)

Reply:
Yeah this might be a conflict miss resolution while merging